modules/simple_forms_api/lib/tasks/send_emails_by_date_range.rake

```diff
@@ -4,60 +4,84 @@
 #   rails "simple_forms_api:send_emails_by_date_range[1 January 2025,2 January 2025]"
 namespace :simple_forms_api do
   task :send_emails_by_date_range, %i[start_date end_date] => :environment do |_, args|
-    start_date, end_date = parse_date_args(args)
+    start_date = Time.zone.parse(args[:start_date])
+    end_date = Time.zone.parse(args[:end_date])
```

Comment:
I'm only pulling this out because the line length is getting too big overall for the linter.